suites/squid/common/sanity/01-rbd.yaml

```diff
@@ -0,0 +1,248 @@
+tests:
+
+# Tests from - suites/squid/rbd/tier-1_rbd.yaml
+
+  - test:
+      name: Install rbd-nbd
+      desc: Install rbd-nbd and remove any epel packages
+      module: exec.py
+      config:
+        sudo: true
+        commands:
+          - "rm -rf /etc/yum.repos.d/epel*"
+          - "dnf install rbd-nbd -y"
+
+  - test:
+      name: 1_rbd_cli_groups
+      desc: "Executing upstream RBD CLI Groups scenarios"
+      polarion-id: CEPH-83574239
+      module: test_rbd.py
+      config:
+        script: rbd_groups.sh
+        script_path: qa/workunits/rbd
+
+  - test:
+      name: 2_rbd_cli_import_export
+      desc: "Executing upstream RBD CLI Import Export scenarios"
+      polarion-id: CEPH-83574240
+      module: test_rbd.py
+      config:
+        script: import_export.sh
+        script_path: qa/workunits/rbd
+
+  - test:
```

Comment:
http://magna002.ceph.redhat.com/cephci-jenkins/results/openstack/RH/8.0/rhel-9.5/Sanity/19.2.0-55/28/01-rbd/4_rbd_cli_migration_0.log
It needs empty pools in that cluster as we are performing all test in one cluster it may not be possible, let's remove this test

Reply:
@sunilangadi2 won't it affect sanity coverage ? Can't it takes care in test case ?

Reply:
it's an upstream test so we can't do anything. it's ok it will be covered in the upstream pipeline.

Reply:
@sunilangadi2 thanks, removed.